packages/block-library/src/file/block.json

```
@@ -70,7 +70,10 @@
 				"link": true
 			}
 		},
-		"interactivity": true
+		"interactivity": true,
+		"interactivity": {
+			"clientNavigation": true
+		}
```

This one is already interactive.

```suggestion
		"interactivity": true
```

---

And was duplicated, my bad. Similar for search and navigation.